packages/project-disaster-trail/src/state/chapters.js

```diff
@@ -3,7 +3,7 @@ import CHAPTERS from "../constants/chapters";
 
 const lastChapterIndex = CHAPTERS.length - 1;
 const initialState = {
-  activeChapterIndex: 0
```

Comment:
I think we should leave this at 0 since it's the true default for the game, different engineers are working on different chapters, and it's easy for us to change it to whichever chapter while developing.

Reply:
Ah yes. Totally.